RestApi/Sepes.RestApi/Model/Rule.cs

```diff
@@ -1,36 +1,34 @@
 using System;
-using System.Collections.Generic;
-using System.Linq;
 using System.Net;
 
 namespace Sepes.RestApi.Model
 {
     public class Rule
     {
-        public readonly ushort port;
-        public readonly IPAddress ip;
+        public ushort port { get; }
+        public string ip { get; }
 
         public Rule(ushort port, IPAddress ip)
         {
             this.port = port;
-            this.ip = ip;
+            this.ip = ip.ToString();
```

Comment:
Might be useful to have a MapToIpv4().ToString() instead. Not sure how it decides if its gonna write ipv4 or ipv6 without that function.